pages/orders/index.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+import { ref } from "vue";
+import Table from "../components/Table/Table.vue";
+import Modal from "../components/Modal/Modal.vue";
```

Comment:
After rename Modal and Table files You can delete this lines:
```
import { ref } from "vue";
import Table from "../components/Table/Table.vue";
import Modal from "../components/Modal/Modal.vue";
```
ref is also autoimported in nuxt